code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx

```diff
@@ -18,61 +29,60 @@ const profileData = {
   phone: '+1 123-456-7890',
 };
 
-
-export const Profile = () => {
+export const Profile: React.FC = () => {
   return (
-    <Box sx={{ width: '100%', maxWidth: { sm: '100%', md: '800px' }, mx: 'auto', p: 3, borderRadius: 2 }}>
-      <CardContent sx={{ textAlign: 'center', mb: 3 }}>
+    <Box className={styles.profileContainer}>
+      <CardContent className={styles.cardContentCentered}>
         <Avatar
-          alt={profileData.firstName + ' ' + profileData.lastName}
-          src="/path/to/avatar.jpg"
-          sx={{ width: 120, height: 120, margin: '0 auto', mb: 2 }}
+          src="/code/packages/web/src/pages/dashboard/dashboard_pages/profile/avatar.jpg"
```

Comment:
Avatar is not being displayed. Plus inline paths can cause errors:

Do this instead:
- Move `avatar.jpg` to `src/assets` folder
- Import img like this `import avatarSrc from 'src/assets/avatar.jpg';`
- Use that here like this `src={avatarSrc}`